setup.py

```diff
@@ -61,7 +61,7 @@
         "gitpython",
         "bluepyopt>=1.14.10",
         "bluepyefe>=2.2.0",
-        "neurom>=3.0,<5.0",
+        "neurom>=3.0",
```

Comment:
Why removing this? In a far future NeuroM v5 might appear :smile:

Reply:
because @darshanmandge forced me to

Reply:
That's a very good reason! :rofl:

Reply:
I hope there will be backward compatibility in the neurom code so that the functions used in bluepyemodel(bpem) keep on working >=3.0 :p . Earlier it was <4.0 and when I upgrade bluepyemodel with new neurom4.0 in my venv, bpem complains.

Reply:
The neurom function used in BPEM are still present in neurom 4, but some has a slightly different behavior.

Reply:
So, the optimisation results will differ when user upgrade bluepyemodel with this code?

Reply:
If they are adding a calcium hotspot, yes. If they are not adding a calcium hotspot, no.

Reply:
Ok. I think till now only we have used those function. So, it is fine.